core/domain/takeout_service_test.py

```diff
@@ -94,7 +96,8 @@
     models.Names.SUBTOPIC,
     models.Names.SUGGESTION,
     models.Names.TOPIC,
-    models.Names.USER
+    models.Names.USER,
+    models.Names.OPPORTUNITY
```

Comment:
Why isn't this following alphabetical order?

Reply:
Removed it.